packages/eslint-plugin/test/test-utils.ts

```diff
@@ -0,0 +1,4 @@
+export const espreeParser: string = require('resolve-from')(
```

Comment:
Because it uses a commonjs export (`export =`) which `@babel/plugin-transform-typescript` doesn't support out of the box, and this codebase doesn't have `esModuleInterop` enabled.

[I have written a babel plugin that adds support for `export =` that could be used](https://www.npmjs.com/package/babel-plugin-replace-ts-export-assignment), but generally `esModuleInterop` is the better way to go and since this is for tests I thought it would be simpler to just use `require` here than to muck about with the babel config.

Reply:
Yeah we should eventually enable `esModuleInterop`... I've added that to my personal todo list. What you did is totally fine for now.

Reply:
would it be problematic to enable `esModuleInterop` as part of this PR? if it is - then don't do it but if it's just a matter for using that option then this could be included here

Reply:
The trouble is that `esModuleInterop` actually changes the emitted code so in theory it could result in a breaking change depending on the structure and usage, for users that are not using a transpiler like Babel or TypeScript.

It's unlikely, but I'd prefer to keep this PR to be focused on just `eslint-plugin` if possible since enabling `esModuleInterop` doesn't give a strong win here.